subscription_oca/models/sale_subscription.py

```diff
@@ -464,7 +501,7 @@ def create(self, values):
                 values["date_start"] = values["recurring_next_date"]
             values["stage_id"] = (
                 self.env["sale.subscription.stage"]
-                .search([("type", "=", "pre")], order="sequence desc", limit=1)
+                .search([("type", "=", "draft")], order="sequence desc", limit=1)
```

Comment:
I think this is not longer needed with the addition of the _get_default_stage_id method

Reply:
And i also think that the first conditional of the create function could be removed because it is unreachable code